README.md

```diff
@@ -54,275 +55,250 @@ limitations under the License.
-
-First, create a virtual environment with the version of Python you're going to use and activate it.
-
-**macOS/Linux**
-
-```python -m venv env
-source env/bin/activate
-```
-
-**Windows**
-
-``` python -m venv env
-env\Scripts\activate
-```
+<a target="_blank" href="https://huggingface.co/enterprise">
+    <img alt="Hugging Face Enterprise Hub" src="https://github.com/user-attachments/assets/247fb16d-d251-4583-96c4-d3d76dda4925">
+</a><br>
 
-To use 🤗 Transformers, you must install at least one of Flax, PyTorch, or TensorFlow. Refer to the official installation guides for platform-specific commands:
+## Why shouldn't I use Transformers?
 
-[TensorFlow installation page](https://www.tensorflow.org/install/), 
-[PyTorch installation page](https://pytorch.org/get-started/locally/#start-locally) and/or [Flax](https://github.com/google/flax#quick-install) and [Jax](https://github.com/google/jax#installation) 
+- This library is not a modular toolbox of building blocks for neural nets. The code in the model files is not refactored with additional abstractions on purpose, so that researchers can quickly iterate on each of the models without diving into additional abstractions/files.
+- The training API is optimized to work with PyTorch models provided by Transformers. For generic machine learning loops, you should use another library like [Accelerate](https://huggingface.co/docs/accelerate).
+- The [example scripts]((https://github.com/huggingface/transformers/tree/main/examples)) are only *examples*. They may not necessarily work out-of-the-box on your specific use case and you'll need to adapt the code for it to work.
 
-When one of those backends has been installed, 🤗 Transformers can be installed using pip as follows:
+## 100 projects using Transformers
 
-```
-pip install transformers
-```
+Transformers is more than a toolkit to use pretrained models, it's a community of projects built around it and the
+Hugging Face Hub. We want Transformers to enable developers, researchers, students, professors, engineers, and anyone
+else to build their dream projects.
 
-If you'd like to play with the examples or need the bleeding edge of the code and can't wait for a new release, you must [install the library from source](https://huggingface.co/docs/transformers/installation#installing-from-source).
+In order to celebrate Transformers 100,000 stars, we wanted to put the spotlight on the
+community with the [awesome-transformers](./awesome-transformers.md) page which lists 100
+incredible projects built with Transformers.
 
-```
-git clone https://github.com/huggingface/transformers.git
-cd transformers
-pip install .
-```
+If you own or use a project that you believe should be part of the list, please open a PR to add it!
 
-### With conda
+## Example models
```

Comment:
Let me know if you think we're missing a task or we should highlight a different model!